project.clj

```diff
@@ -85,8 +94,11 @@
                                   [prismatic/schema-generators "0.1.0"
                                    :exclusions [prismatic/schema]]]
                    :resource-paths ["model"
-                                    "test/resources"]}
-
+                                    "test/resources"]
+                   :jvm-opts ["-Dcom.sun.management.jmxremote"
+                              "-Dcom.sun.management.jmxremote.ssl=false"
+                              "-Dcom.sun.management.jmxremote.authenticate=false"
+                              "-Dcom.sun.management.jmxremote.port=5555"]}
```

Comment:
What is JMX for?  Can we put it into a different profile, so that it can be added on demand?